applications/GeoMechanicsApplication/custom_utilities/transport_equation_utilities.hpp

```diff
@@ -13,6 +13,7 @@
 #pragma once
 
 // Project includes
+#include <optional>
```

Comment:
I believe that with your latest changes this `#include` has become redundant.

Reply:
Sorry, removed.